src/main/java/com/google/firebase/database/logging/LogWrapper.java

```diff
@@ -18,19 +18,36 @@
 
 import java.io.PrintWriter;
 import java.io.StringWriter;
+import org.slf4j.LoggerFactory;
 
-/** User: greg Date: 6/12/13 Time: 2:23 PM */
+/**
+ * Legacy logging interface for database implementation. This class attempts to reconcile SLF4J
+ * with the old logging implementation of Admin SDK. When SLF4J is available, logs using that
+ * API. Otherwise falls back to the old logging implementation. This class will removed in a
```

Comment:
I assume we could just log to both but we're avoiding that because it would potentially result in duplicate logs for people?  Perhaps worth spelling that out in the comment.